### ISSUE TITLE
Can't install latest beta 0.3.1? 

### ISSUE THREAD

**Issue by Skrell** [yoshiask/FluentStore#47]
### Before you start...

- [X] Have you checked if similar issue have already been reported?

### What windows are you seeing the problem on?

Windows 10

### What OS/App version/architecture are you seeing the problem on?

x64

### Description

When I double click either of the files *.appinstaller or *msixbundle I get a message saying "The app package is not supported for installation by App Installer because it uses restricted capabilities"? 

### Steps To Reproduce

Double click either installer. 

### Expected behavior

Installation to complete.

**Comment by SleepDaemon** [comment]
I get the same issue as well, version 2 still works mainly cause its an .exe

![image](https://user-images.githubusercontent.com/88533953/180655935-52686c41-7e68-4e03-a414-93a36ad986da.png)

**Comment by yoshiask** [comment]
It seems like the 0.3.1 workaround for the startup crash is causing issues on machines that don't have Developer Mode enabled. I have a fix in mind.

Just out of curiosity, what happens if you run this PowerShell command in the folder you downloaded the App Installer?
```
Add-AppxPackage -AppInstallerFile "<path to the App Installer file you downloaded>"
```

**Comment by Skrell** [comment]
I actually enabled developer mode and it made no difference.

**Comment by Skrell** [comment]
> It seems like the 0.3.1 workaround for the startup crash is causing issues on machines that don't have Developer Mode enabled. I have a fix in mind.
> 
> Just out of curiosity, what happens if you run this PowerShell command in the folder you downloaded the App Installer?
> 
> ```
> Add-AppxPackage -AppInstallerFile "<path to the App Installer file you downloaded>"
> ```

Add-AppxPackage : An invalid app installer file name was passed to this function. This file must have the
.appinstaller extension.
At line:1 char:1
+ Add-AppxPackage -AppInstallerFile ".\FluentStoreBeta_0.3.1.msixbundle ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-AppxPackage], ArgumentException
    + FullyQualifiedErrorId : System.ArgumentException,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCo
   mmand

**Comment by Skrell** [comment]
Just realized I didn't follow your instructions correctly, this is what happens when I user the *.appinstaller
![image](https://user-images.githubusercontent.com/590266/180666658-43c6365b-5a3e-406a-af18-deb37229763f.png)

**Comment by yoshiask** [comment]
@Skrell Does the PowerShell command complete? The greenish part should be a progress bar.

**Comment by Skrell** [comment]
Nope it sits there forever unfortunately.

**Comment by Skrell** [comment]
I had no issue installing the previous version if that helps.... :-/

**Comment by yoshiask** [comment]
> I had no issue installing the previous version if that helps.... :-/ 

Yeah, that's expected. As I said in an earlier comment, the fix for the 0.3.0 startup crash is exactly what's causing the issue here. I'll have to undo that and fix it in some other way.

**Comment by yoshiask** [comment]
@Skrell Has this issue been resolved with v0.3.2-beta?

**Comment by Skrell** [comment]
Yep it installed!!  Thank you!

**Comment by SleepDaemon** [comment]
> @Skrell Has this issue been resolved with v0.3.2-beta?

I got this on 0.3.2
![image](https://user-images.githubusercontent.com/88533953/181528916-b5558854-58dc-4943-aecc-6da906c47592.png)

**Comment by yoshiask** [comment]
> > @Skrell Has this issue been resolved with v0.3.2-beta?
> 
> I got this on 0.3.2
> ![image](https://user-images.githubusercontent.com/88533953/181528916-b5558854-58dc-4943-aecc-6da906c47592.png)
> 

@SleepDaemon Is this with the App Installer or the MSIX bundle? Either way, can you try the App Installer again?

**Comment by yoshiask** [comment]
I'm closing this issue. If you're still having problems installing, please file a new report.
